vtysh/vtysh_main.c

```diff
@@ -535,6 +535,8 @@ int main(int argc, char **argv, char **env)
 	/* Do not connect until we have passed authentication. */
 	if (vtysh_connect_all(daemon_name) <= 0) {
 		fprintf(stderr, "Exiting: failed to connect to any daemons.\n");
+		fprintf(stderr,
+			"Hint: if this seems wrong, try running me as a privileged user!\n");
```

Comment:
Looks useful to me. You could also add an `if (geteuid() != 0)` check to avoid showing this message when we know vtysh was started by a privileged user. What do you think?

Reply:
I was just thinking about that, sounds fine.